code/datalab-app/src/app/components/welcome/NavBar.js

```diff
@@ -1,22 +1,67 @@
 import React from 'react';
-import { Container, Button, Menu } from 'semantic-ui-react';
+import { withStyles } from 'material-ui/styles';
+import { grey, blueGrey, teal } from 'material-ui/colors';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import Button from 'material-ui/Button';
+import classNames from 'classnames';
 import { NavLink } from 'react-router-dom';
+import datalabsLogo from '../../../assets/images/datalabs-hori.png';
 import auth from '../../auth/auth';
 
 const publicNavLinks = [
-  { label: 'DataLabs', path: '/', exact: true },
-  { label: 'About', path: '/about', exact: true },
+  { name: 'Slack' },
+  { name: 'Discourse' },
+  { name: 'GitHub' },
+  { name: 'DockerHub' },
 ];
 
-const PublicNavBarContent = () => (
-  <Container>
-    <Menu inverted pointing secondary>
-      {publicNavLinks.map(props => (<NavLink className="item" exact={props.exact} key={props.label} to={props.path}>{props.label}</NavLink>))}
-      <Menu.Menu position="right">
-      <Button primary onClick={auth.login}>Login</Button>
-      </Menu.Menu>
-    </Menu>
-  </Container>
+const styles = theme => ({
+  appBar: {
+    // boxShadow: 'none',
+  },
+  toolbar: {
+    paddingLeft: '8%',
+    paddingRight: '4%',
+    width: '100%',
+    padding: 0,
+    backgroundColor: blueGrey[900],
+  },
+  logo: {
+    width: 140,
+  },
+  spacer: {
+    flex: 1,
```

Comment:
Needs to be checked in IE. Use of flex is pretty much a no-no still I think....sadly.